internal/builds/builds_test.go

```diff
@@ -22,24 +24,25 @@ func TestBuildsIntegrationSuite(t *testing.T) {
 	suite.Run(t, new(BuildsIntegrationTestSuite))
 }
 
-func (suite *BuildsIntegrationTestSuite) SetupSuite() {
-	suite.app = initTestApp(suite.T())
-	// ensure the db is clean before running suite
-	testutils.Cleanup(suite.T(), suite.app.db)
+func (suite *BuildsIntegrationTestSuite) SetupTest() {
+	// start a new db transaction for each test
```

Comment:
Maybe this comment should reference that this is an effect of initTestApp specifically--it wasn't until I looked there that I understood what was going on here, I thought this comment was misplaced